sklearn/utils/setup.py

```diff
@@ -54,6 +55,24 @@ def configuration(parent_package='', top_path=None):
                          include_dirs=[numpy.get_include()],
                          libraries=libraries)
 
+    # generate files from a template
+    pyx_templates = ['sklearn/utils/seq_dataset.pyx.tp',
+                     'sklearn/utils/seq_dataset.pxd.tp']
+
+    for pyxfiles in pyx_templates:
+        outfile = pyxfiles[:-3]
```

Comment:
small comment: I find `.replace('.tp', '')` slightly more readable.

Reply:
I should take mental note of this one.